piet-svg/src/lib.rs

```diff
@@ -119,7 +120,8 @@ impl piet::RenderContext for RenderContext {
                     gradient.append(
                         svg::node::element::Stop::new()
                             .set("offset", stop.pos)
-                            .set("stop-color", fmt_color(&stop.color)),
+                            .set("stop-color", fmt_color(&stop.color))
+                            .set("stop-opacity", fmt_opacity(&stop.color)),
```

Comment:
Should we only specify `stop-opacity` (and above, `fill-opacity`) for non-opaque colors?

Reply:
I would keep it as it. It would reduce the resulting svg output size but the codes get's a bit more complex, I don't think there is an particular advantage of one over the other